### PR TITLE
styled navbar and added filter buttons

### PR DESCRIPTION
**Pull request by Aryanshhandelwal-systango**

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import './header.css';
+import cartIcon from '../../assets/Icon-04 1.svg';
+import profileIcon from '../../assets/Icon-03 1.svg';
+import searchIcon from '../../assets/Icon-02 1.svg';
+import logo from '../../assets/logo.png'
+import {NavLink} from 'react-router-dom';
+import {Container, Row} from 'reactstrap';
+
+
+const nav__links = [
```
Comment (rohit-systango) [inline src/components/Header/Header.jsx]: @Aryanshhandelwal-systango  use proper variable declaration format here

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      <Container>
+        <Row>
+          <div className='nav_wrapper'>
+            <div className='logo'>
+              <img src={logo} alt='logo' />
+            </div>
+
+            <div className='navigation'>
+              <ul className='menu'>
+               {nav__links.map((item,index) => (
```
Comment (rohit-systango) [inline src/components/Header/Header.jsx]: @Aryanshhandelwal-systango  use optional  chaining for best practices

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+               </span>
+              <span className='cart_icon'>
+              <img src={cartIcon} alt='cartIcon' />
+              <span className="badge">1</span>
+              </span>
+            </div>
+          </div>
+          <div className='invite_box'>
+            <div className='invite_content'>
+            <p className='invite_line'>Invite friends to Fashion Festival & get up to $150 Bonus for every Referral</p>
```
Comment (rohit-systango) [inline src/components/Header/Header.jsx]: @Aryanshhandelwal-systango have messages file , where you put all the static value which will be directly rendered in html

Comment (rohit-systango) [reply src/components/Header/Header.jsx]: check in whole PR for the same

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+          {/*main-page-content*/}
+          <span className='small-text'>
+            <p className='small-text'>Home / Clothing / Mens Clothing / All Mens Clothing</p>
+          </span>
+          <span className='details'>
+            <p className='details'><b>All Products</b> (25 Products) </p>
+          </span>
+          <span className='filter'>
+            <h4>Filters:</h4>
+            <button className='filter_buttons'>T-shirt</button>
```
Comment (rohit-systango) [inline src/components/Header/Header.jsx]: @Aryanshhandelwal-systango use map here and make list of buttons and iterate over it

File: src/routers/Routers.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import {Routes, Route, Navigate} from 'react-router-dom';
+
+import Home from '../pages/Home';
+import Cart from '../pages/Cart';
+import ProductDetails from '../pages/ProductDetails';
+
+const Routers = () => {
+  return (
+   <Routes>
+   <Route path='/' element= {<Navigate to='home' />} />
```
Comment (rohit-systango) [inline src/routers/Routers.js]: @Aryanshhandelwal-systango make routes file for managing route

**Review by rohit-systango — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (Aryanshhandelwal-systango) [reply src/components/Header/Header.jsx]: changed

File: src/components/Header/Header.jsx (hunk 4 above)
Comment (Aryanshhandelwal-systango) [reply src/components/Header/Header.jsx]: i have the updated code,just not appending due to some css styles
will append shortly

**Pull request merged**